src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -983,7 +983,9 @@ void goto_instrument_parse_optionst::instrument_goto_program()
        cmdline.isset("custom-bitvector-analysis"))
       config.ansi_c.defines.push_back("__CPROVER_CUSTOM_BITVECTOR_ANALYSIS");
 
-    status() << "Adding CPROVER library" << eom;
+    // add the library
+    status() << "Adding CPROVER library ("
+             << config.ansi_c.arch << ")" << eom;
```

Comment:
As there are even more users of `link_to_library`: the status message should really just become part of the `link_to_library` implementation rather than replicating the code. Then the status() output should be removed at each callee.